### ISSUE TITLE
baseline results

### ISSUE THREAD

**Issue by mingkai-zheng** [Beckschen/TransMix#12]
I just checked the configuration file, it seems that some of the training strategies are quite different from the original DeiT training recipe,  (e.g. Batch size, learning rate scheduler, model ema ...) So I'm wondering what would be the baseline result for this configuration?
